### ISSUE TITLE
Update multiple entity managers

### ISSUE THREAD

**Issue by sergiu-popa** [symfony/symfony-docs#8980]
Can we update this article for SF4? https://symfony.com/doc/current/doctrine/multiple_entity_managers.html

This is the new default configurations for doctrine:

```
doctrine:
    dbal:
        # configure these for your database server
        driver: 'pdo_mysql'
        server_version: '5.7'
        charset: utf8mb4

        # With Symfony 3.3, remove the `resolve:` prefix
        url: '%env(resolve:DATABASE_URL)%'
```

This is an example of multiple connections using the url option:

```
doctrine:
    dbal:
        # configure these for your database server

        # With Symfony 3.3, remove the `resolve:` prefix
        default_connection: default

        connections:
            default:
                url: '%env(resolve:DATABASE_URL)%'
                driver: 'pdo_mysql'
                server_version: '5.7'
                charset: utf8mb4
            books:
                url: '%env(resolve:DATABASE_URL_BOOKS)%'
```

**Comment by javiereguiluz** [comment]
Thanks for reporting this and sorry it took us so long to respond. We're fixing it in #10093.

**Comment by sergiu-popa** [comment]
Better late than never! I also do this, replying to things months old,
because I keep track of stuff left undone. So thanks!

**Comment by javiereguiluz** [comment]
Fixed by #10093.
